### PR TITLE
Vale and markdownlint changes for docs/specs.

### PR DESCRIPTION
**Pull request by cgcardona**

### CODE REVIEW

File: .github/styles/Vocab/Products/accept.txt [hunk 1]
```diff
@@ -253,6 +257,7 @@ subcommands
 subgraph
 Subgraph
 Subnet
+SUBNET
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:260]: This should not be included.

File: .github/styles/Vocab/Products/accept.txt [hunk 2]
```diff
@@ -270,6 +275,8 @@ toolchain
 Toolchains
 trie
 TXIDs
+Tx
+TX
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:279]: Not sure about these two. Should we make TX the only option?

File: .github/styles/Vocab/Products/accept.txt (hunk 1 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:260]: How should we handle the following line from https://raw.githubusercontent.com/ava-labs/avalanche-docs/master/docs/subnets/subnet-upgrade.md?

```
:::warning NETWORK UPGRADES PERMANENTLY CHANGE THE RULES OF YOUR SUBNET.
```

Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:260]: Just make it not all caps?

File: .github/styles/Vocab/Products/accept.txt (hunk 2 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:278]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 1 above)
Comment (yulin-dong) [reply .github/styles/Vocab/Products/accept.txt:260]: Yeah, `Network Upgrades Permanently Change the Rules of Your Subnet.`

File: docs/subnets/subnet-upgrade.md [hunk 3]
```diff
@@ -17,14 +17,16 @@ to alter the way Subnet runs, for various business or operational needs. These u
 the purview of your team, and you have complete control over the timing of their roll out. Any such
 change represents a **network upgrade** and needs to be carefully planned and executed.
 
-:::warning NETWORK UPGRADES PERMANENTLY CHANGE THE RULES OF YOUR SUBNET.
+:::warning 
+NETWORK UPGRADES PERMANENTLY CHANGE THE RULES OF YOUR SUBNET.
```
Comment (yulin-dong) [inline docs/subnets/subnet-upgrade.md:21]:
```suggestion
Network Upgrades Permanently Change the Rules of Your Subnet.
```

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**